### PR TITLE
FIX: Attr conf send null when should true

### PR DESCRIPTION
**Pull request by anitatomaszewska**

### CODE REVIEW

File: src/Divante/MagentoIntegrationBundle/Infrastructure/Common/AbstractIntegratedObjectRepository.php [hunk 1]
```diff
@@ -75,7 +75,7 @@ public function getObjects(array $objectsId, IntegrationConfiguration $configura
                 ]
             )
         );
-
+        $listing->setObjectTypes([AbstractObject::OBJECT_TYPE_VARIANT, AbstractObject::OBJECT_TYPE_OBJECT]);
```
Comment (synchmann) [inline src/Divante/MagentoIntegrationBundle/Infrastructure/Common/AbstractIntegratedObjectRepository.php:80]: Also needs to me added in function `getAllObjects`

File: src/Divante/MagentoIntegrationBundle/Application/Product/MappedProductService.php [hunk 2]
```diff
@@ -46,6 +46,7 @@ public function getProducts(string $ids, string $instanceUrl, string $storeViewI
             ];
         }
         $configuration = reset($configurations);
+        AbstractObject::setGetInheritedValues(true);
```
Comment (synchmann) [inline src/Divante/MagentoIntegrationBundle/Application/Product/MappedProductService.php]: I think it should be added in integratedObjectRepository function `getObjects`, so in future implementation we will no have to think about it when fetching objects

Comment (anitatomaszewska) [reply src/Divante/MagentoIntegrationBundle/Application/Product/MappedProductService.php]: It doesn't work as it should, wen we put this into getObjects method.

File: src/Divante/MagentoIntegrationBundle/Infrastructure/Common/AbstractIntegratedObjectRepository.php (hunk 1 above)
Comment (anitatomaszewska) [reply src/Divante/MagentoIntegrationBundle/Infrastructure/Common/AbstractIntegratedObjectRepository.php:80]: added

**Pull request merged**